sys-apps/policycoreutils/policycoreutils-3.1-r3.ebuild

```
@@ -219,7 +219,7 @@ pkg_postinst() {
 	for POLICY_TYPE in ${POLICY_TYPES} ; do
 		# There have been some changes to the policy store, rebuilding now.
 		# https://marc.info/?l=selinux&m=143757277819717&w=2
-		einfo "Rebuilding store ${POLICY_TYPE} (without re-loading)."
-		semodule -s "${POLICY_TYPE}" -n -B || die "Failed to rebuild policy store ${POLICY_TYPE}"
+		einfo "Rebuilding store ${POLICY_TYPE} in '${ROOT}' (without re-loading)."
+		semodule -S "${ROOT}" -s "${POLICY_TYPE}" -n -B || die "Failed to rebuild policy store ${POLICY_TYPE}"
```

I think that ROOT can be empty if root is actually `/`, so I think we should guard against this case too (if we want to upstream this to gentoo too).

```suggestion
		einfo "Rebuilding store ${POLICY_TYPE} in '${ROOT%/}' (without re-loading)."
		semodule -S "${ROOT%/}" -s "${POLICY_TYPE}" -n -B || die "Failed to rebuild policy store ${POLICY_TYPE}"
```

---

Eeek, I got it wrong, should be:
```suggestion
		einfo "Rebuilding store ${POLICY_TYPE} in '${ROOT:-/}' (without re-loading)."
		semodule -S "${ROOT:-/}" -s "${POLICY_TYPE}" -n -B || die "Failed to rebuild policy store ${POLICY_TYPE}"
```

---

See the [second implication in the ultimate EAPI7 guide](https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#d-ed-root-eroot-no-longer-have-a-trailing-slash).

---

Good catch Krzesimir! I'll go with it.